mythril/interfaces/cli.py

```diff
@@ -275,6 +278,22 @@ def main():
                 "The --query-signature function requires the python package ethereum-input-decoder",
             )
 
+    if args.enable_iprof:
+        ignore_iprof = True
+
+        if args.v < 4:
+            print(
```

Comment:
I see other places in the command line option handling are using `print`. Besides, if I use `log`, which log level should I use? `log.warn/error/info` will be masked if `-v` is too low.

Or does it not matter a warning message is not displayed?

Reply:
Using print is fair. I guess a question now is do we want to exit immediately because an invalid combination of flags was used, rather than just continuing and providing a message that might be lost in debug output?

Reply:
Agree, exit immediately makes more sense. I'll change in the next version.